integration-tests/rita.py

```diff
@@ -341,7 +339,7 @@ def read_email(node):
             mail = json.load(mail_file_handle)
             if mail["envelope"]["forward_path"][0] == "{}@example.com".format(id):
                 return ''.join(chr(i) for i in mail["message"])
-    print("cannot find email for node {}".format(id))
+    raise Exception("cannot find email for node {}".format(id))
```

Comment:
Why use an exception? How is it any better than `teardown()` and then `sys.exit(1)`?